requirements.txt

```diff
@@ -1,2 +1,2 @@
 GitPython==3.0.2
-PyGithub==1.43.8
\ No newline at end of file
+PyGithub==1.43.8
```

Comment:
Please could you squash your commits. That will remove this diff which does nothing and make the overall change a bit cleaner.

Reply:
Hi, thanks for the response, I am new to contributing to open source projects. I hope I did it right.

Reply:
No worries. 😄  It doesn't look right though. This PR now includes a number of commits that were made by me recently. I think your fork was some commits behind. Not sure how to fix it now so it might be best to re-fork the repo and resubmit a PR. Sorry for the trouble.

Reply:
Scratch that. I think I can salvage this by cherry picking your commit from a temporary branch.